terraform/modules/logging/log-metric-filters/main.tf

```diff
@@ -285,7 +285,7 @@ resource "aws_cloudwatch_log_metric_filter" "request_time_bucket_router_8" {
 
 resource "aws_cloudwatch_log_metric_filter" "request_time_bucket_router_9" {
   name           = "${var.environment}-router-request-times-9"
-  pattern        = "{$$.requestTime >= 10 && $$.request != \"*/_status?ignore-dependencies *\"}"
+  pattern        = "{$$.requestTime >= 10 && $$.request != \"*/_status?ignore-dependencies *\" && $$.request != \"POST*\" && $$.size < 500000}"
```

Comment:
Will this include a slow POST with a small request size?

Reply:
all conditions operated by &&, so all four must be true to send the alarm

Reply:
I think we probably want to know about small slow POSTs. I'd be happy to remove the `!= POST` condition.

Reply:
I can also separate the two conditions with an || and put them into brackets.